src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -462,6 +465,19 @@ public T WithTelemetry(ITelemetryConfig telemetryConfig)
             return (T)this;
         }
 
+        /// <summary>
+        /// Sets the parameters required to get a Kerberos Ticket from Azure AD service.
+        /// </summary>
+        /// <param name="servicePrincipalName">Service principal name to get Kerberos Service Ticket.</param>
+        /// <param name="ticketContainer">Container to use for Kerberos Ticket.</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public T WithKerberosTicketClaim(string servicePrincipalName, KerberosTicketContainer ticketContainer)
```

Comment:
With this, Kerberos will be made available to all flows of public client applications and confidential client applications. Do we really support this for confidential client apps?

Reply:
Actually from your wiki it seems this is only available for public client flows.... So it should go into `PublicClientApplicationBuilder`